docs/protocol.md

```diff
@@ -949,6 +953,122 @@ Response (in case of success)
 ```
 </details>
 
+### Spreading a Rumor
+
+🧭 **RPC Message** > **Query** > **Rumor**
+
+By sending a Rumor message, a server propagates to all Servers in the network messages it encapsulates. 
+
+In order to propagate quicker messages across the network, a Rumor message would be used upon receiving a new message that should be spread. This would trigger the Gossip network protocol and rumor would be relayed according to it. When receiving a new Rumor, Servers process contained messages if they are new.
```

Comment:
```suggestion
In order to propagate messages faster across the network, a Rumor message would be used upon receiving a new message that should be spread. This would trigger the Gossip network protocol and rumor would be relayed according to it. When receiving a new Rumor, Servers process the contained messages if they are new.
```